tests/acceptance/sonar-project.properties

```diff
@@ -0,0 +1,54 @@
+sonar.projectKey=owncloud-1_ocis_acceptance-tests
+sonar.organization=owncloud-1
+sonar.projectVersion=7.1.1
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/ocis
+sonar.links.ci=https://drone.owncloud.com/owncloud/ocis/
+sonar.links.scm=https://github.com/owncloud/ocis
+sonar.links.issue=https://github.com/owncloud/ocis/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# SQ standard properties
+sonar.sources=.
+
+# Pull Requests
+sonar.pullrequest.provider=github
+sonar.pullrequest.github.repository=owncloud/ocis
+sonar.pullrequest.base=${env.SONAR_PULL_REQUEST_BASE}
+sonar.pullrequest.branch=${env.SONAR_PULL_REQUEST_BRANCH}
+sonar.pullrequest.key=${env.SONAR_PULL_REQUEST_KEY}
+
+# Properties specific to language plugins:
+sonar.go.coverage.reportPaths=results/coverage-*.out
+
+# Exclude files
+sonar.exclusions=**/third_party,docs/**,changelog/**,**/package.json,**/rollup.config.js,CHANGELOG.md,deployments/**,tests/**,vendor/**,vendor-bin/**,README.md,**/mocks/**,/protogen/**,**/*_gen.go
+sonar.coverage.exclusions=**/*_test.go,**mocks/**,/protogen/**,**/*_gen.go
+sonar.cpd.exclusions=**/defaultconfig.go,**/*_test.go,**/revaconfig/**,services/settings/pkg/store/defaults/defaults.go
+
+# Rule exclusions
+sonar.issue.ignore.multicriteria=g1,g2
+
+# Ignore "Define a constant instead of duplicating this literal" rule for tests
+sonar.issue.ignore.multicriteria.g1.ruleKey=go:S1192
+sonar.issue.ignore.multicriteria.g1.resourceKey=**/*_test.go
+
+# Ignore "Rename function XXX to match the regular expression ^(_|[a-zA-Z0-9]+)$" rule for tests
+sonar.issue.ignore.multicriteria.g2.ruleKey=go:S100
+sonar.issue.ignore.multicriteria.g2.resourceKey=**/*_test.go
```

Comment:
I suppose that we need to ignore "code quality" errors because this Sonar run is only for gathering code coverage.
The "regular" Sonar run that goes with unit tests be the one that we care about for "code quality" feedback.

Reply:
This was just a comment "to myself" to explain why we ignore these SonarCloud "code quality" errors for this special sonar-project.